LKQL_scripts.rst

```diff
@@ -142,7 +142,7 @@ LKQL Script::
 
     let result = query t @ *
                     when isRealWithoutRange(t) ||
-                        t is SubtypeDecl(any superTypes: s@_ when isRealWithoutRange(s))
+                        t is SubtypeDecl(any super_types: s@_ when isRealWithoutRange(s))
```

Comment:
Shouldn't this be
any super_types is s@* when ...

Reply:
You're right! TBH I should just delete this file, it's pretty obsolete by now ...